nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors/src/main/java/org/apache/nifi/processors/solr/SolrUtils.java

```diff
@@ -149,6 +151,25 @@
             .required(false)
             .build();
 
+    public static final PropertyDescriptor KERBEROS_PRINCIPAL = new PropertyDescriptor.Builder()
+            .name("Kerberos Principal")

```

Comment:
Minor issue but I would use `.name("kerberos-principal")` and `.displayName("Kerberos Principal")` here.